sources/packages/web/src/components/common/AssessmentAwardDetails.vue

```diff
@@ -0,0 +1,233 @@
+<template>
+  <div v-if="assessmentAwardData.estimatedAward">
+    <!-- Show the first payment/second payment header only if there is second disbursement. -->
+    <h3
+      v-if="isSecondDisbursementAvailable"
+      class="category-header-medium secondary-color my-3"
```

Comment:
mm, is it not brand black?? 
![image](https://user-images.githubusercontent.com/77353155/191812826-00eb73d1-42d4-48ed-a25f-00509280ec83.png)